Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/PayPalTokenizationViewModel.swift

```diff
@@ -15,21 +15,19 @@ class PayPalTokenizationViewModel: PaymentMethodTokenizationViewModel, ExternalP
     private var orderId: String?
     private var confirmBillingAgreementResponse: PayPalConfirmBillingAgreementResponse?
     
-    override lazy var title: String = {
-        return "PayPal"
-    }()
+    private lazy var _title: String = { return "PayPal" }()
+    override var title: String  {
+        get { return _title }
+        set { _title = newValue }
+    }
     
-    override lazy var buttonTitle: String? = {
-        switch config.type {
-        case .payPal:
-            return nil
-        default:
-            assert(true, "Shouldn't end up in here")
-            return nil
-        }
-    }()
+    private lazy var _buttonTitle: String? = { return nil }()
+    override var buttonTitle: String? {
+        get { return _buttonTitle }
+        set { _buttonTitle = newValue }
+    }
```

Comment:
all the overridden are going to be `internal`?

Reply:
Yeah, all of those are only used internally.